src/middlewared/middlewared/plugins/directoryservices_/secrets.py

```diff
@@ -188,12 +188,8 @@ async def backup(self):
         )
 
     async def restore(self, netbios_name=None):
-        failover_status = await self.middleware.call('failover.status')
-        if failover_status not in ('SINGLE', 'MASTER'):
-            self.logger.debug("Current failover status [%s]. Skipping secrets restore.",
-                              failover_status)
-            return False
-
+        """ Restore the contents of secrets.tdb from a stored backup of the node.
+        This is allowed if standby controller as it preps winbindd for failover. """
```

Comment:
'if' sounds odd, maybe
```suggestion
        This is allowed on standby controller as it preps winbindd for failover. """
```